spec/models/portfolio_item_spec.rb

```diff
@@ -69,6 +69,26 @@
         subject.run_callbacks kind
       end
     end
+
+    it "validate_deletable callback" do
+      expect(subject).to receive(:validate_deletable)
+
+      subject.run_callbacks 'destroy'
+    end
+  end
+
+  context "#validate_deletable" do
+    let(:portfolio) { create(:portfolio) }
+    subject! { create(:portfolio_item, :portfolio => portfolio) }
+
+    it "when no associated order processes" do
+      expect(subject.validate_deletable).to be_nil
+    end
+
+    it "when has associated order processes" do
```

Comment:
should have a `context` block for the when condition and `it` block for the result. `tag_add` can be in the before block